src/components/AddTransaction.js

```diff
@@ -14,28 +14,40 @@ export const AddTransaction = () => {
       id: Math.floor(Math.random() * 100000000),
       text,
       amount: +amount
-    }
+    };
 
     addTransaction(newTransaction);
-  }
+    text = "";
```

Comment:
Can you actually do this? 
```suggestion
    setText("");
    setAmount(0);
```

Reply:
Yeah!